### DISCUSSION TITLE
Possibility to schedule the execution of a workflow manually with a delay of X seconds

### DISCUSSION THREAD

**Discussion by rojo1997** [argoproj/argo-workflows#12151] [Ideas]
I think it could be interesting to be able to schedule the execution of any workflow with a certain delay. I don't refer to the possibility of creating a cronjob, but to have a box to specify the number of seconds of delay and that by default it is 0. Currently, if I'm not mistaken, the only way to have this capability is to run a container with a sleep seconds parameter.

**Comment by agilgur5** [comment]
Duplicate of #10620 / #7201, which has one open proposal as well in #11559
